graph.go

```diff
@@ -55,22 +57,33 @@ var graphCmd GraphCmd
 var allowErrorsInGoGet = true
 
 func (c *GraphCmd) Execute(args []string) error {
-	var unit *unit.SourceUnit
-	if err := json.NewDecoder(os.Stdin).Decode(&unit); err != nil {
+	inputBytes, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
 		return err
 	}
+	var units unit.SourceUnits
+	if err := json.NewDecoder(bytes.NewReader(inputBytes)).Decode(&units); err != nil {
+		// Legacy API: try parsing input as a single source unit
+		var unit *unit.SourceUnit
+		if err := json.NewDecoder(bytes.NewReader(inputBytes)).Decode(&unit); err != nil {
+			return err
+		}
+		units = append(units, unit)
```

Comment:
Also, since `units` was declared as type `unit.SourceUnits`, wouldn't it be fitting to do `units = unit.SourceUnits{unit}` rather than `units = []*unit.SourceUnit{unit}`?